packages/app/src/debug/DebugRunNavigationLimitMessage.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="flex text-sm p-4 align-center">
+    <div class="mr-2 pt-[2px]">
+      <IconWarningCircle
+        stroke-color="gray-500"
+        fill-color="gray-50"
+      />
+    </div>
+    <div class="text-gray-700">
+      {{ t('debugPage.foundMoreThan100Runs') }}
+      <ExternalLink
+        v-if="runUrl"
+        :href="cloudProjectHref"
+      >
+        {{ t('debugPage.goToCypressCloud') }}
+      </ExternalLink>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue'
+import ExternalLink from '@cy/gql-components/ExternalLink.vue'
+import { useI18n } from '@cy/i18n'
+import { IconWarningCircle } from '@cypress-design/vue-icon'
+
+const { t } = useI18n()
+
+const props = defineProps<{runUrl: string | null}>()
+
+const cloudProjectHref = computed(() => {
+  return props.runUrl?.slice(0, props.runUrl.indexOf('/runs'))
```

Comment:
I'm making some assumptions about the URL in order to derive the project URL from the run URL

Reply:
Fixed in https://github.com/cypress-io/cypress/pull/26523/commits/3c57c26032bd11de085f49815b53fb4693d12f4f